build/cloud/07-deployment.yaml

```diff
@@ -28,13 +28,13 @@ spec:
         - /bin/sh
         - -c
         - |
-          apk --no-cache add coreutils && cat | tee /etc/kubeedge/cloud/kubeconfig.yaml <<EOF
+          cat | tee /etc/kubeedge/cloud/kubeconfig.yaml <<EOF
           apiVersion: v1
           kind: Config
           clusters:
           - name: kubeedge
             cluster:
-              certificate-authority-data: $(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | base64 -w 0)
+              certificate-authority-data: $(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | base64 | awk '{printf $0}')
```

Comment:
Any reason we don't use simply `certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt` ? I mean actually there is no reason to put the ca into the kubeconfig since in all containers the ca will be mounted at ` /var/run/secrets/kubernetes.io/serviceaccount/ca.crt`

Reply:
Thank @johscheuer ,use simply certificate-authority works fine here. So we don't need base64( or any other advanced functions in coreutils) anymore.